data/modules/CargoRun/CargoRun.lua

```diff
@@ -0,0 +1,1138 @@
+-- Copyright © 2008-2014 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local Engine = import("Engine")
+local Lang = import("Lang")
+local Game = import("Game")
+local Space = import("Space")
+local Comms = import("Comms")
+local Event = import("Event")
+local Mission = import("Mission")
+local NameGen = import("NameGen")
```

Comment:
`NameGen` isn't used, so you can just remove it. (An inheritance from the copy-paste from DeliverPackage.)